js/ui.js

```
@@ -5569,6 +5741,14 @@ async function handleUIClicks(e) {
       setCustomLocation(true)
       break;
     }
+    case "manage-locations": {
```

Biome (2.5.6)

[error] Duplicate case label.

(lint/suspicious/noDuplicateCase)